src/gtest/test_foundersreward.cpp

```diff
@@ -106,11 +106,11 @@ TEST(founders_reward_test, general) {
     // address = t2ENg7hHVqqs9JwU5cgjvSbxnT2a9USNfhy
     // script.ToString() = OP_HASH160 55d64928e69829d9376c776550b6cc710d427153 OP_EQUAL
     // HexStr(script) = a91455d64928e69829d9376c776550b6cc710d42715387
-    EXPECT_EQ(params.GetFoundersRewardScriptAtHeight(1), ParseHex("a914ef775f1f997f122a062fff1a2d7443abd1f9c64287"));
+    EXPECT_EQ(HexStr(params.GetFoundersRewardScriptAtHeight(1)), "a914ef775f1f997f122a062fff1a2d7443abd1f9c64287");
```

Comment:
what does this have to do with the prevector?

Reply:
Good catch, the change to this file should be dropped.

Reply:
There is no implementation of equality now after the other changes, so this change is necessary or it won't compile.

Reply:
I don't see an equality operator for CScript being erased in this commit or the previous one.
So does that mean some commits in this PR don't compile?

Reply:
I guess what's going on is when there's a ToString method it can compare as strings and that was erased in the prevector commit. So the third commit doesn't compile, and the fourth does https://github.com/zcash/zcash/pull/3180/commits/d2fb34fb7ca56e612bfc619224aa27cd914aeb55
I guess that's fine.

Reply:
It's not that it was erased, just that upstream didn't implement it. The change here is definitely a no-op, as comparing two hex strings should always be the same as comparing their decoded forms.

Reply:
Provided the case matches, and ``HexStr`` always produces lowercase so that should be fine.